CHANGES

```diff
@@ -68,6 +68,8 @@ Universes
   module and library boundaries. Global universe names introduced in an
   inductive / constant / Let declaration get qualified with the name of
   the declaration.
+- Fix BZ#5726: Notations that start with `Type` now support universe instances
```

Comment:
I think you didn't understand what @maximedenes was suggesting. We used `BZ#` for bug reports from the Bugzilla time to prevent against any ambiguity with pull requests with the same number. But in this case, the bug number is high enough so that there is no ambiguity and you can simply refer to it as #5726. Sorry about the nitpicking.

Reply:
Ah, got it, fixed. No problem, I don't mind some work for consistency (though documentation on this kind of thing is nice to have somewhere).

Reply:
Thanks. I was thinking that after some time, just looking at what is done in the file would be enough to know what to do but as you are one of the first ones to mention a bug number since the migration, you are missing these examples. If this kind of mistake happens too often, I guess it will be a reason to document it more properly. Also, you are welcome to preemptively open PRs or issues to improve on what you feel is currently unclear.